app/modules/parsing/graph_construction/parsing_repomap.py

```
@@ -615,7 +615,8 @@ def create_graph(self, repo_dir):
         seen_relationships = set()
 
         for root, dirs, files in os.walk(repo_dir):
```

Ruff (0.14.13)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)